src/EntityFramework6.Npgsql/EntityFramework6.Npgsql.nuspec

```diff
@@ -16,7 +16,7 @@
     <language>en-US</language>
     <tags>npgsql postgresql postgres data database entity framework ef orm</tags>
     <dependencies>
-      <dependency id="Npgsql" version="3.1.0" />
+      <dependency id="npgsql" version="3.1.0" />
       <dependency id="EntityFramework" version="6.0.0" />
```

Comment:
I didn't do that because I thought that if you specify 3.1.0, Nuget will get the latest minor version automatically, whereas if I specify 3.1.5 explicitly, it will be directly linked to that minor version and won't bring down 3.1.6 or later.

Since the new package name is present since 3.1.5, it will always get 3.1.5 or newer, so I think this is safe to leave as-is.

I'm not familiar with Nuget version semantics, so I might be wrong, but in my own fork it seemed to do as it should and I'm using it in a .NET core project right now.